Pilot/FileLoader.swift

```diff
@@ -29,19 +30,25 @@ class FileLoader: NSObject {
     for path in directoryPath {
       let file = File<NSDictionary>(path: path)
 
-      // Make a new PilotFile based on the file locaiton
-      let pilotFile = PilotFile(name: file.name, size: (file.size?.hashValue)!, thumbnail: NSImage(named: "DocumentIcon"), directory: file.path.description, writeTime: NSData().description, id: "4")
+      // Strip the extension from the file name
+      let fileName: NSString = file.name
```

Comment:
It has to be of type NSString to use the stringByDeletingPathExtention method. File class has no such method.